meetings/2022-01-26.md

```diff
@@ -0,0 +1,97 @@
+* Wouter: While I do agree that the questions about legally enforcing policies are not in the scope of the panel, is there a place where we could dive in the boundaries between the technical and legal aspect?
+* Elf: I'm not sure, but I'm also interested.
+* Wouter: Maybe we can look into that.
+* Tom: I've thought about that a lot in the past and the goals in the pannels is to provide something that is privacy compliant. It might be hard to enforce, but at least providing adequate tooling for the willing, that would be great. Maybe a data-protection panel? It might not be an issue on the technical level, but on the user level.
+* Elf: Maybe we could start talking about this in the interop panel. It seems that the themes overlap. Access/Consent/Data grants are in the same space. I'd like to propose it as a next item on the interop panel.
+* Matthieu: I think maybe there *is* on technical aspect about it, i.e. qua data modeling what we should use (e.g. ODRL). Maybe in such a data-protection panel we could research and decide on such a common way of non-technically enforcable rules.
+* Tom: I've also been working on that for the last few years. We need to distinguish giving access to data from giving consent to data. Almost all organisations will process your data based on consent. Giving access to data is not the same to giving consent. A company given consent can process your data. Giving access is sometimes less permissive. A company or a bank might be legally obliged to keep and process data about you regardless of your consent. In addition you have to keep in mind that these legal obligations are subject to change. Access and consent are related, but not the same. We should focus on access control, not on legal reasons.
+* Mathieu: Now that you mention it, these legal reasons will also differ from (legal) framework to framework.
+* Tom: Exactly and Matthieu's consent should not be conflated with consent from a legal perspective.
+* Matthieu: When it comes from a user, we try to use "consent" to indicate "from the will of the user", but maybe "access" is a more neutral term.
+* Tom: Yes, for example, permits or permission could work, but I would avoid using the word consent.
+* Wouter: I think last week in the interop panel, the difference between access and consent was mentioned. Sharing information in an access consent registry would provide a way for us to inform legal concerns.
+* Elf: The name access consent was mostly driven by the fact it's derived from the "consent screen" on the application. I'm open to renaming it. But if you look at the application, it can be broad enough to encompass data you have access to coming from different resource owners. It is setting a policy about data coming from different resource owners. But you cannot go beyond single storage. That's the kind of issue we address. It can issue data grants that are more granular and could output ACLs. https://github.com/solid/authorization-panel/discussions/281
```

Comment:
```suggestion
* Wouter: I think last week in the interop panel, the difference between "access" and "consent" was mentioned. Sharing information in an "access consent" registry would provide a way for us to inform legal concerns.
* Elf: The name "access consent" was mostly driven by the fact it's derived from the "consent screen" on the application. I'm open to renaming it. But if you look at the application, it can be broad enough to encompass data you have access to coming from different resource owners. It is setting a policy about data coming from different resource owners. But you cannot go beyond single storage. That's the kind of issue we address. It can issue data grants that are more granular and could output ACLs. https://github.com/solid/authorization-panel/discussions/281
```